new_tutorials/02_build_a_scalable_question_answering_system.ipynb

```diff
@@ -0,0 +1,502 @@
+{
```

Comment:
In light of my overall comment, if this is _replacing_ our current tutorial 2 (which I don't think it is, that's a fine tuning one), make sure the aliases are in place or if it's replacing tutorial 3 (Basic QA pipeline without elastic search) then the name should be `03_Build_A_Sclable_Question_Answering _System` and the old one should be added to the `index.toml` as an alias..

Reminder that the numbering in the name isn't linked to the order they show up as on tutorials. So if you end up eventually wanting this to be the second to show up as it is with what I think you're replacing it with: **https://haystack.deepset.ai/tutorials/03_basic_qa_pipeline_without_elasticsearch**
You can just set the `weight` in `index.toml` accordingly.

Making sure the file names, numbering is correct already will save some work at the end when we agree on the changes I think 🙌🏽